package.json

```diff
@@ -6,6 +6,7 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
+    "circleci": "npm run test -- --no-watch",
```

Comment:
Did you try if this is working? `ng test` has only `watch=true|false` as per reference: https://angular.io/cli/test

Reply:
Yes, it was working, but I fixed it